libs/commons/Hashtbl_.mli

```diff
@@ -6,3 +6,32 @@ type 'a hashset = ('a, bool) Hashtbl.t
 
 val hashset_of_list : 'a list -> 'a hashset
 val hashset_to_list : 'a hashset -> 'a list
+
+(* Safe replacement for Hashtbl_.get_stack for OCaml < 5
```

Comment:
you mean Safe replacement for Hashtbl.find_all

Reply:
This is one of the dangers of global replace (with sed, not semgrep).